test/gtest/stringify.cpp

```diff
@@ -335,14 +335,55 @@ TEST_F(StringifyTest, FilterLocalSets) {
   auto substrings = repeatSubstrings(stringify.hashString);
   auto result = StringifyProcessor::dedupe(std::move(substrings));
 
-  result = StringifyProcessor::filter(
-    std::move(substrings), stringify.exprs, [](const Expression* curr) {
-      return curr->is<LocalSet>();
-    });
+  result =
+    StringifyProcessor::filterLocalSets(std::move(substrings), stringify.exprs);
 
   EXPECT_EQ(
     result,
     (std::vector<SuffixTree::RepeatedSubstring>{
       // sequence i32.const 0, i32.const 1 appears at idx 6 and again at 16
       SuffixTree::RepeatedSubstring{2u, (std::vector<unsigned>{6, 16})}}));
 }
+
+TEST_F(StringifyTest, FilterBranches) {
+  static auto branchesModuleText = R"wasm(
+  (module
+  (func $a (result i32)
```

Comment:
It would be good to indent everything inside the module.
```suggestion
  (module
   (func $a (result i32)
```

Reply:
done!